crates/forge_analyzer/src/permissionclassifier.rs

```diff
@@ -0,0 +1,280 @@
+use crate::checkers::IntrinsicName;
+use forge_loader::forgepermissions::ForgePermissions;
+
+pub(crate) fn check_permission_used(
+    function_name: IntrinsicName,
+    first_arg: &str,
+    second_arg: Option<&str>,
+) -> Vec<ForgePermissions> {
+    let mut used_permissions: Vec<ForgePermissions> = Vec::new();
+
+    let post_call = second_arg.unwrap_or(&String::from("")).contains("POST");
```

Comment:
```suggestion
    let post_call = second_arg.unwrap_or("").contains("POST");
```

You can also have `second_arg` just be a `&str` and pass in an empty string instead of `None`.